src/main/resources/pirk.properties

```diff
@@ -31,7 +31,8 @@ local.pirk.properties=/root/local.pirk.properties
 ##
 ##Spark path for SparkLauncher
 ##
-spark.home = /usr
+#spark.home = /usr
+spark.home=/opt/cloudera/parcels/CDH/lib/spark
```

Comment:
That's my intermediate scratch - I will remove once I remove the WIP from the pull request (I was running the distributed testing on a CDH cluster)